raw_data_QA/get_data.sh

```diff
@@ -0,0 +1,8 @@
+rm -r data
+mkdir data
+curl -L https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-counties.csv > data/us-counties-latest.csv
```

Comment:
I'm not sure we need this on an on-going basis. Today I added snapshotting of nytimes data in the public repo, here: https://github.com/covid-projections/covid-data-public/tree/master/data/cases-nytimes

Reply:
Yes, this was more for ad-hoc reviews.

For automated checks, it would be great to add that as an option and link to the snapshosts you take (especially if they match the API snapshots). Are your snapshots in sync with the API snapshots? If so, I could further develop this to link to your snapshots.